src/Iot.Device.Bindings/Iot.Device.Bindings.csproj

```diff
@@ -8,6 +8,7 @@
     <DeviceRoot>$(MSBuildThisFileDirectory)../devices/</DeviceRoot>
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);PackageRefAssembly</TargetsForTfmSpecificContentInPackage>
+	  <LangVersion>8</LangVersion>
```

Comment:
Seems like you have a tab here. Also, we already did this in a different PR so you don't need to do it any longer.